### PR TITLE
feat: added cli to packages

### PR DESCRIPTION
**Pull request by drorrose**
@davidmeirlevy 
After npm install

### CODE REVIEW

**Review by davidmeirlevy — APPROVED** [review]

**Comment by davidmeirlevy** [comment]
I think we should maybe change the CLI so it won't be a part of the general installation, because it makes the install of the actual project slower..
Maybe create a folder called "tools" and put the cli in it (you can call the folder just "cli" and not "greenpress-cli").

**Comment by drorrose** [comment]
@davidmeirlevy 
Moved it into tools, please check it out. Will merge after your approval :)

**Review by davidmeirlevy — APPROVED** [review]

**Pull request merged**
